custom/gcc_sangath/rules/custom_actions.py

```diff
@@ -76,7 +76,7 @@ def _get_case_updates(peer_rating_cases):
 
 def _get_sum(case_property, peer_rating_cases):
     return sum([
-        int(peer_rating_case.get_case_property(case_property) or 0)
+        float(peer_rating_case.get_case_property(case_property) or 0.0)
```

Comment:
nit: I think the change to 0.0 isn't needed, passing `0` to float would handle that for you and is easier to read.

Reply:
✅